src/oas.md

```diff
@@ -557,6 +557,31 @@ servers:
 
 An object representing a Server Variable for server URL template substitution.
 
+The server URL templating is defined by the following [ABNF](https://tools.ietf.org/html/rfc5234) syntax.
+
+```abnf
+server-url-template  = 1*( literals / server-variable )
+server-variable      = "{" server-variable-name "}"
+server-variable-name = 1*( %x00-79 / %x7C / %x7E-10FFFF ) ; every UTF8 character except { and }
```

Comment:
This is incorrect as it excludes the `z` character. Whenever the server variable name contains `z` the ABNF fails.
Here is a quick suggestion to fix this temporarily including `z` explicitly. This also affects **ABNF** grammar for Path Templating in the same way.

```suggestion
server-variable-name = 1*( %x00-79 / %x7C / %x7E-10FFFF / "z" ) ; every UTF8 character except { and }
```

Reply:
```suggestion
server-variable-name = 1*( %x00-7A / %x7C / %x7E-10FFFF ) ; every UTF8 character except { and }
```

Reply:
ooooh thanks for catching that! this is probably a case of me not seeing properly and skipping a line in the ASCII table. I'll also correct the path templating one in a minute.

Reply:
np. FYI for path templating I've issued a separate fixing PR in https://github.com/OAI/OpenAPI-Specification/pull/4279